tests/local/test_ScraperExtractFromTextTest.py

```diff
@@ -9,6 +9,8 @@ class ScraperExtractFromText(unittest.TestCase):
     without a full integration test.
     """
 
+    # To avoid AssertionError when comparing example results for Colorado Appellate
+    maxDiff = None
```

Comment:
I commented this out and saw no problem.  Are you sure this is needed?

Reply:
No anymore.
The first example I used was very long and raised the `AssertionError` error. The example in PR doesn't need it.